### DISCUSSION TITLE
pntfx.com

### DISCUSSION THREAD

**Discussion by fabiangabel** [thuliteio/doks#926] [Showcase]
We used the doks theme to build an application to visualize and learn the connections between mathematical concepts:

https://pntfx.com

> Mathematics is not about isolated formulae but about understanding concepts and connections. In pntfx ('bridge-builder' (pons + facere)), we focus on this network of mathematics!

Source code available at https://github.com/HOOU-TUHH/Pontifex-Hugo

Let us know what you think :-)

![image](https://user-images.githubusercontent.com/3385756/196306427-85a99664-c867-4056-ac32-a0003945801e.png)

**Comment by h-enk** [comment]
Wow. Great site, I especially love the way you can navigate through graphs.

I think this is a great example of what you can make with Doks! Are you ok with me adding [pntfx](https://pntfx.com/) to the Doks showcase?

**Reply by fabiangabel** [reply]
@h-enk Thank you very much for your feedback! You are very welcome to add pntfx to the Doks showcase :-)

**Reply by h-enk** [reply]
I've added pntfx to the Doks Showcase: https://getdoks.org/showcase/pntfx/

**Reply by fabiangabel** [reply]
This looks great. Thank you very much @h-enk !
